PubSub/sdk/src/main/java/org/lantern/pubsub/AutoStart.java

```diff
@@ -0,0 +1,19 @@
+package org.lantern.pubsub;
```

Comment:
For the Android app, I'm actually following the Sun convention and using the inverse of our domain for the package name: ``org.getlantern.lantern`` But I like ``org.lantern``, if only because it's shorter; the ``getlantern`` is unnecessary. I think after this is merged we could rename it to be consistent.